editoast/src/views/sub_categories.rs

```
@@ -160,7 +171,21 @@ async fn delete_sub_category(
         return Err(AuthorizationError::Forbidden.into());
     }
 
-    let mut conn = db_pool.get().await?;
+    let conn = db_pool.get().await?;
+
+    let _ = conn
+        .transaction(|mut tx| {
+            Box::pin(async move { delete_sub_category_and_fallback_to_main(&mut tx, code).await })
+        })
+        .await;
```

Errors should not be silently ignored